### ISSUE TITLE
Have you tested the results on other models and datasets? e.g. cifar and wideresnet GN?

### ISSUE THREAD

**Issue by Lily-Le** [mr-eggplant/SAR#10]
I tried to pretrain a widerenet28x10 on cifar10 with groupnorm, and the final ACC on clean dataset is about 93%. 
Then I employed sar for adaptation on cifar10-c severity5. The adaptation error is about 

[23/07/27 22:28:16] [cifar10c_group_sar.py:  109]: error % [gaussian_noise5]: 57.30%  
[23/07/27 22:29:29] [cifar10c_group_sar.py:  109]: error % [shot_noise5]: 51.10%  
[23/07/27 22:30:40] [cifar10c_group_sar.py:  109]: error % [impulse_noise5]: 65.13%  
[23/07/27 22:31:52] [cifar10c_group_sar.py:  109]: error % [defocus_blur5]: 30.20%  
[23/07/27 22:33:05] [cifar10c_group_sar.py:  109]: error % [glass_blur5]: 44.16%  
[23/07/27 22:34:18] [cifar10c_group_sar.py:  109]: error % [motion_blur5]: 21.98%  
[23/07/27 22:35:29] [cifar10c_group_sar.py:  109]: error % [zoom_blur5]: 28.73%  
[23/07/27 22:36:42] [cifar10c_group_sar.py:  109]: error % [snow5]: 17.81%  
[23/07/27 22:37:54] [cifar10c_group_sar.py:  109]: error % [frost5]: 22.26%  
[23/07/27 22:39:06] [cifar10c_group_sar.py:  109]: error % [fog5]: 19.91%  
[23/07/27 22:40:19] [cifar10c_group_sar.py:  109]: error % [brightness5]: 8.45%  
[23/07/27 22:41:30] [cifar10c_group_sar.py:  109]: error % [contrast5]: 27.89%  
[23/07/27 22:42:43] [cifar10c_group_sar.py:  109]: error % [elastic_transform5]: 27.14%  
[23/07/27 22:43:55] [cifar10c_group_sar.py:  109]: error % [pixelate5]: 33.00%  
[23/07/27 22:45:03] [cifar10c_group_sar.py:  109]: error % [jpeg_compression5]: 30.21%  
[23/07/27 22:45:03] [cifar10c_group_sar.py:  117]: error % [mean5]: 32.35%  

Is that normal? Have you tested the results on other models and datasets? Thanks so much.

**Comment by mr-eggplant** [comment]
Hi, I am not sure about that as we did not test sar on WideResnet GN + Cifar-10-C. 
But one point to be noticed is that the hyper-parameters may need careful tuning for your setting, such as learning rate.
